### PR TITLE
README Revamp

### PR DESCRIPTION
**Pull request by akalicki**
Addresses #6 - rewrote readme to regurgitate less text from the original PlanOut release and focus more on implementation differences + how the package is used at Vimeo. cc @eytan for merge.

### CODE REVIEW

**Comment by akalicki** [comment]
I'm sure you're super busy but any feedback @eytan? Tried to wrap in some info to make this different than the original PlanOut README so hopefully it's improved a bit!

File: README.md [hunk 1]
```diff
@@ -146,7 +80,92 @@ repository, we recommend including it as a project dependency in your
...
+
+[logging traits]: https://github.com/vimeo/ABLincoln/tree/master/src/Vimeo/ABLincoln/Experiments/Logging
+[FileLoggerTrait]: src/Vimeo/ABLincoln/Experiments/Logging/FileLoggerTrait.php
+[PSRLoggerTrait]: src/Vimeo/ABLincoln/Experiments/Logging/PSRLoggerTrait.php
+[Monolog Handler]: https://github.com/Seldaek/monolog
+
+##### Traffic Segmentation
+
+What if you want to deploy a test comparing users from a specific location or
+demographic to the overall population? Vimeo refers to this process as traffic
```
Comment (eytan) [inline README.md]: When you run an experiment you only want to compare outcomes under different treatments for equivalent populations.  This first sentence doesn't sound compatible with this.  I am not sure what Vimeo does, but one should only exposure log people who pass the gating conditions. It is then only equivalent users who have passed the gating checks that are compared in the analysis of the experiments. Individuals with the "default parameter" group should not be logged.

File: README.md [hunk 2]
```diff
@@ -7,84 +7,24 @@
 
 ABLincoln is a PHP-based toolkit for online field experimentation, ported from
 Facebook's [PlanOut][] software and released as an easily extendable
-[Composer][] package. ABLincoln makes it easy to deploy and maintain
-sophisticated experiments and to quickly iterate on these experiments, while
-satisfying the constraints of large-scale Internet services with many users.
+[Composer][installation] package. ABLincoln makes it easy to deploy and
```
Comment (eytan) [inline README.md]: How about: ABLincoln is a PHP-based implementation of Facebook's PlanOut, a framework for online field experimentation. ?

File: README.md [hunk 3]
```diff
@@ -146,7 +80,92 @@ repository, we recommend including it as a project dependency in your
 ```
 
 [Composer]: https://getcomposer.org/
-[Packagist]: https://packagist.org/
+[Packagist]: https://packagist.org/packages/vimeo/ablincoln
+
+### Comparison with Python Release
+
```
Comment (eytan) [inline README.md]: Comparison with the PlanOut reference implementation

File: README.md [hunk 4]
```diff
@@ -146,7 +80,92 @@ repository, we recommend including it as a project dependency in your
...
+[PlanOut language]: http://facebook.github.io/planout/docs/planout-language.html
+
+### Usage in Production Environments
+
+ABLincoln was ported and designed with scalability in mind. Here are a few ways
+that Vimeo has chosen to extend it to meet the needs of our testing process:
+
+##### Serialized Experiments
+
```
Comment (eytan) [inline README.md]: It doesn't appear that any of this functionality is included in the current release so I don't think it makes sense to include it in the README...

Comment (akalicki) [inline README.md]: @jacoboliver time to get working on our blog post!

File: README.md (hunk 1 above)
Comment (akalicki) [inline README.md]: That makes sense. No longer have access to Vimeo's codebase so I can't say for sure that's what we did, but I'll remove this section for now.

File: README.md (hunk 2 above)
Comment (akalicki) [inline README.md]: :+1:

**Comment by akalicki** [comment]
Made a few changes, if you're ok with how it looks I'll merge it in the morning!

**Comment by eytan** [comment]
LGTM, great work alex. I would check out the first few sentences of the current about page of fb for a few more ideas on the acknowledgements section, which isn't quite accurate as it stands. 

I definitely owe you a dinner when you come by to the Bay Area this summer!

**Comment by akalicki** [comment]
Looking forward to it! :smile:

**Pull request merged**